change/@internal-react-composites-a3db1ef9-13e9-4688-9dc9-fb533e4192a8.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
can you regenerate this changelog? Looks to have not changelog text in it

Reply:
lol I was wondering what happened there 😅